spi/verifiable-credential-spi/src/main/java/org/eclipse/edc/identityhub/spi/verifiablecredentials/CredentialRequestService.java

```diff
@@ -22,13 +22,19 @@
  * Makes credential requests to a given issuer
  */
 public interface CredentialRequestService {
+    /**
+     * Service type for entries in a DID document that contain the fully-qualified Credential Request API endpoint
+     */
+    String CREDENTIAL_REQUEST_SERVICE_TYPE = "CredentialRequest";
```

Comment:
I think the issuer entry is `IssuerService` in the DID document

Reply:
ah true. and it also is supposed to be just the base URL, without the `/credentials` path suffix. will adapt.